modules/apps/segments/segments-web/src/main/java/com/liferay/segments/web/internal/display/context/SegmentsExperienceSelectorDisplayContext.java

```diff
@@ -48,38 +49,74 @@
 public class SegmentsExperienceSelectorDisplayContext {
 
 	public SegmentsExperienceSelectorDisplayContext(
-		HttpServletRequest httpServletRequest,
-		SegmentsExperienceManager segmentsExperienceManager) {
+		HttpServletRequest httpServletRequest, JSONFactory jsonFactory,
+		Language language, SegmentsEntryLocalService segmentsEntryLocalService,
+		SegmentsExperienceManager segmentsExperienceManager,
+		SegmentsExperienceLocalService segmentsExperienceLocalService,
+		SegmentsExperimentLocalService segmentsExperimentLocalService,
+		SegmentsExperimentRelLocalService segmentsExperimentRelLocalService,
+		Portal portal) {
```

Comment:
We should sort parameters alphabetically, so I think Portal should be declared sooner.